src/main/java/racingcarwinner/Car.java

```diff
@@ -0,0 +1,41 @@
+package racingcarwinner;
+
+public class Car {
+
+    private Position position;
+    private final Name name;
+    public Car(String name){
+        this.position = new Position(0);
+        this.name = new Name(name);
+
+    }
+    public Position getPosition() {
+        return position;
+    }
+
+    public Name getName(){
+        return this.name;
+    }
+
+    public void move() {
+        position = new Position(position.getIncreaseValue());
+    }
```

Comment:
```suggestion
    public void move() {
        position.move();
    }
```
이런식으로 표현할 수 있게 만들면 어떨까요?